sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -136,6 +136,10 @@ class ParquetFileFormat
         s"Set Parquet option ${ParquetOutputFormat.JOB_SUMMARY_LEVEL} to NONE.")
     }
 
+    conf.set(
+      SQLConf.LEGACY_PARQUET_REBASE_DATETIME.key,
```

Comment:
Can we use `SQLConf.get` in the place we need to read the config directly? It can simplify the code.

Reply:
We can but I just follow the style of the method. See lines above.